connector/connectionProperties.js

```diff
@@ -2,9 +2,20 @@
     var props = {};
     props["user"] = attr[connectionHelper.attributeUsername];
     props["password"] = attr[connectionHelper.attributePassword];
+    props["authSource"] = attr["v-authDatabase"];
+    props["database"] = attr[connectionHelper.attributeDatabase];
     props["dialect"] = "mongosql";
-    props["database"] = attr[connectionHelper.attributeDatabase]
-    props["authSource"] = "admin";
+    props["TYPE_MAPPING_MODE"] = "tableau";
```

Comment:
```suggestion
```
I don't think this property exists in the JDBD driver, does it?

Reply:
It is in the design for the type mapping, I will remove it for now.